test/spec/escapesequences.js

```diff
@@ -648,7 +648,7 @@
           "init": [
             {
               "type": "StringLiteral",
-              "value": "'",
+              "value": "\\'",
```

Comment:
This is a breaking change.

Or to be fair, it would be, if `.value` had been set properly in the first place, which, as seems, it didn't: escapes like `\b` and `\n` are interpreted in `.value`, but `\x` aren't. And so `"\65"` and "A" have different `.value`s despite representing the same string. And `\x` escapes weren't supported before Lua 5.2 anyway. Oh well.

I guess decimal and hex character escapes should be interpreted too. That would make the most sense.